keyboards/hotdox/rules.mk

```
@@ -0,0 +1,94 @@
+#----------------------------------------------------------------------------
+# On command line:
+#
+# make = Make software.
+#
+# make clean = Clean out built project files.
+#
+# That's pretty much all you need. To compile, always go make clean,
+# followed by make.
+#
+# For advanced users only:
+# make teensy = Download the hex file to the device, using teensy_loader_cli.
+#               (must have teensy_loader_cli installed).
+#
+#----------------------------------------------------------------------------
+
+# # project specific files
+SRC =  matrix.c     \
+	   twimaster.c  \
```

Please remove this and replace it with i2c_master.c, which is the global implementation of this. 

```suggestion
	   i2c_master.c  \
```